src/data/general-cs-quiz.ts

```diff
@@ -1881,7 +1881,52 @@ const generalCSQuiz = [
     Explanation:
       "NTFS, which stands for NT file system and the New Technology File System, is the file system that the Windows NT operating system (OS) uses for storing and retrieving files on hard disk drives (HDDs) and solid-state drives (SSDs). It is the default file system for the Windows NT family and it is developed by Microsoft.",
     Link: "https://en.wikipedia.org/wiki/NTFS"
-  }
+  },
+  {
+    Question: "What is the primary purpose of the Domain Name System (DNS) in networking?",
+    Answer: "To translate human-readable domain names to IP addresses",
+    Distractor1: "To encrypt data transmitted over the network",
+    Distractor2: "To manage and allocate network bandwidth",
+    Distractor3: "To monitor network traffic for security threats",
+    Explanation: "The Domain Name System (DNS) is responsible for converting domain names, which are easy for humans to remember, into IP addresses that computers use to identify each other on the network.",
+    Link: "https://www.cloudflare.com/learning/dns/what-is-dns/"
```

Comment:
We only allow links from freecodecamp sources like the news publication or snippets from freecodecamp youtube videos. we also accept links from official documentation.

we do not accept links from other resources like cloudflare. please use another source.

Reply:
Thanks for the feedback, I have replaced the link with an article from the freecodecamp website.